lib/chorde/shmemutils.py

```diff
@@ -384,7 +384,7 @@ def _value(self):
 if numpy is not None:
 
     class SharedCounterBaseNumpyCtypes(SharedCounterGenericBase):
-        btype = numpy.bool8
+        btype = numpy.bool_
```

Comment:
/ask "are there any other uses of bool8 that need to be fixed outside of what was changed in this PR?"